app/controllers/templates_controller.rb

```diff
@@ -99,6 +99,10 @@ def export
     send_data @template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @template.filename
   end
 
+  def resource_class
```

Comment:
need resource_class to not be private so the index view can check if the to_csv method is supported.